ofxconnect/Makefile.am

```diff
@@ -23,9 +23,9 @@ EXTRA_DIST = cmdline.ggo test-privateserver.sh CMakeLists.txt
 
 # See README.privateserver for details on this server and how to get
 # the key needed to run this test.
-TESTS = test-privateserver.sh 
+TESTS = test-privateserver.sh
 
-ofxconnect.1: $(top_srcdir)/configure.ac
```

Comment:
I understand from your commit message that the missing dependency to `ofxconnect` breaks the parallel build (which is obviously something I didn't test here). 

However, introducing the dependency again will then break the target `make distcheck`, which is why I just removed it in 76dae192 last weekend. If you introduce this dependency, then run `make distcheck`, it eventually fails with

```
ERROR: files left in build directory after distclean:
./ofxdump/ofxdump.1
./ofxconnect/ofxconnect.1
make[1]: *** [Makefile:826: distcleancheck] Fehler 1
```
From https://www.gnu.org/software/automake/manual/html_node/Errors-with-distclean.html I got the impression the error at distcheck is avoided by dropping this dependency. If you can suggest a different change, feel free to adapt your PR.

Reply:
Ah, right. Let me take a look.